src/pages/ForgotPassword.tsx

```
@@ -1,36 +1,146 @@
+  const handleNext = async() => {
+    if (activeStep === 1) {
+      if (!email) {
+        toast.error("Email is required");
+        return;
+      }
+  
+      setIsLoading(true); 
+      const model: validateModel = { email };
+  
+      try {
+        const response: any = await validateAccount({ data: { payload: model } }).unwrap();
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)